css/layout.css

```diff
@@ -0,0 +1,65 @@
+/* layout.css */
+body {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  background-color: #f0f0f0;
+}
+
+header {
+  background-color: #d32f2f;
+  color: white;
+  padding: 1rem;
+  text-align: center;
+}
+
+nav {
+  display: flex;
+  justify-content: space-around;
+  margin-top: 1rem;
+}
+
+nav button {
+  background-color: #f44336;
+  border: none;
+  color: white;
+  padding: 0.5rem 1rem;
+  margin: 0 0.5rem;
+  cursor: pointer;
+}
+
+nav button:hover {
+  background-color: #e53935;
+}
+
+main {
+  flex: 1;
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+section {
+  display: none;
+  width: 100%;
+}
+
+section.active {
+  display: block;
+}
```

Comment:
Rather than doing it this way, you should have a `hidden` class that is `display: none` and you can toggle having `hidden` on or off. This will make it more clear when you want an element to be showing or not.